pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
 
-- Show process(es) that using socket:
-
-`ss -p`
-
 - Filter TCP sockets by states, only/exclude:
 
 `ss {{state/exclude}} {{bucket/big/connected/synchronized/...}}`
 
-- Filter sockets by address and/or port:
+- Show all TCP sockets connected to the local HTTPS port:
```

Comment:
I'd suggest mentioning 443 in the description, e.g. "to the local HTTPS port (443):", to make it explicit.